samples/MvcSandbox/package.json

```diff
@@ -1,4 +1,5 @@
 {
   "name": "ASP.NET",
+  "private":  true,
```

Comment:
Ahh, second line there: https://git-scm.com/docs/gitignore gitignore won't save you from changes to an existing file. Not sure what the correct thing is here, I'm not sure "--assume-unchanged" works across repos?